tdrs-backend/tdpservice/parsers/validators/test/test_category3.py

```diff
@@ -537,7 +537,7 @@ def test_validate__FAM_AFF__SSN():
     }
     result = category3.validate__FAM_AFF__SSN()(instance, schema)
     assert result.valid is False
-    assert result.error == ('T1: Since Item 1 (family affiliation) is 2 and Item 2 (citizenship status) is 1 or 2, '
+    assert result.error == ('T1: Since Item 1 (family affiliation) is 2 and Item 2 (citizenship status) is 1 - 3, '
```

Comment:
This was at linter limit length 120 characters, just did this range instead of `1 or 2 or 3`